lib/ui/viewer/file/fading_app_bar.dart

```diff
@@ -468,6 +461,9 @@ class FadingAppBarState extends State<FadingAppBar> {
       _logger.warning("Failed to save file", e);
       await dialog.hide();
       showGenericErrorDialog(context: context);
+    } finally {
+      PhotoManager.startChangeNotify();
+      LocalSyncService.instance.checkAndSync();
```

Comment:
I am wondering if we really need to sync files here.. May be Just running `_refreshDeviceFolderCountAndCover` might be good enough so that on device albums are in correct state.

Anyways, after thinking more about it, it's fine i guess. Please add .ignore() at the end.

Reply:
Also, let's only call this force sync when getFIle returns a file. If the File download fails, there's no need to trigger the sync.
Ideally, we should call it when both fileDownload and saveAsset is done without error, but adding a check just for file download might be easy & good enough.

Reply:
We do need to sync for the device album to be in the correct state. Just running `_refreshDeviceFolderCountAndCover` is only updating the thumbnail of the device folder is what I observed while testing.